gitgud/test_operator.py

```diff
@@ -22,6 +24,12 @@ def level():
     return all_skills["1"]["1"]
 
 
+@pytest.fixture
+def content_level():
+    from gitgud.skills import all_skills
+    return all_skills['intro']['welcome']
```

Comment:
I don't know that `welcome` is a content level, but it works for the tests

Reply:
Yeah, it would be nice to have handcrafted levels for testing. I added a TODO there.